Lib/test/test_statistics.py

```diff
@@ -2263,6 +2263,22 @@ def test_special_values(self):
         with self.assertRaises(ValueError):
             geometric_mean([Inf, -Inf])
 
+    def test_mixed_int_and_float(self):
+        # Regression test for b.p.o. issue #28327
+        geometric_mean = statistics.geometric_mean
+        expected_mean = 3.80675
```

Comment:
Nitpick-level comment: with this rounded value, the `assertAlmostEqual` test is a bit lopsided: we're checking that `actual_mean` is in an interval that's roughly `[correct_mean - 9e-6, correct_mean + 9e-7]`, so we're allowing a ten times greater error in one direction than the other.  I'd suggest using a more accurate value here to avoid that problem.
```suggestion
        expected_mean = 3.80675409583932
```